Apps/Encounter/src/Models/EncounterModel.cs

```diff
@@ -89,10 +90,16 @@ public static EncounterModel FromODRClaimModel(Claim model)
         public static List<EncounterModel> FromODRClaimModelList(List<Claim> models)
         {
             List<EncounterModel> objects = new List<EncounterModel>();
-
+            Dictionary<string, EncounterModel> dictEncounterIds =
```

Comment:
We are not using the value from the dictionary. Does it need to be a dictionary?

Reply:
@FuriousLlama what is your suggestion?

Reply:
HashSet, for example:
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1?view=net-5.0

Reply:
also: https://stackoverflow.com/questions/1378819/using-a-hashtable-to-store-only-keys

Reply:
good one

Reply:
NIT:  We should be using tasks/9911 for the branch name now that we're using proper feature branches.